lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,33 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+
+  /// The framework will use this method to subscribe to platform focus changes.
+  ViewFocusChangeCallback? _onPlatformFocusChange;
+  Zone _onPlatformFocusChangeZone = Zone.root;
+  set onPlatformFocusChange(ViewFocusChangeCallback callback) {
+    assert(callback == null, 'onPlatformFocusChange was already binded');
+
+    _onPlatformFocusChange = callback;
+    _onPlatformFocusChangeZone = Zone.current;
+  }
+
+  /// The engine will use this to notify the framework a platform focus change.
+  void onPlatformFocusChange(ViewFocusChange platformFocusChange) {
```

Comment:
This should be private. Otherwise, it will be publicly visible through the `dart:ui` API to the framework and the app. Also, I believe the convention is `void _invokePlatformFocusChange(...)` for methods that are invoked internally.

Also, TIL: methods can shadow setters!

Reply:
> Also, TIL: methods can shadow setters!
I think it was just my mistake LOL. 

> This should be private.
Wait if this is private, can the flutter web counterpart override it?